api-specificatie/openapi.yaml

```diff
@@ -408,18 +334,8 @@ paths:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/404'
         '406':
```

Comment:
In deze endpoint zit geen header parameter. Hier zou de 406 dan toch ook weg kunnen ?

Reply:
Goed punt, pas ik aan en haal ook meteen de 412 weg, die kan hier dan ook niet meer optreden.